web/src/components/storage/EncryptionSettingsPage.tsx

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) [2024-2025] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React, { useState, useRef } from "react";
+import { useNavigate } from "react-router-dom";
+import { ActionGroup, Alert, Checkbox, Content, Form, Stack, Switch } from "@patternfly/react-core";
+import { Page, PasswordAndConfirmationInput } from "~/components/core";
+import { EncryptionMethods } from "~/types/storage";
+import sizingStyles from "@patternfly/react-styles/css/utilities/Sizing/sizing";
```

Comment:
I'd like to drop this and its usage, but need a bit more of time for testing how to limit the width used by the Checkbox description without affecting to the rest of the form and allowing `Form#isWidthLimited` to do its work properly.

Will invest some time on Monday, if time permits.